src/jit/stacklevelsetter.cpp

```diff
@@ -151,6 +153,10 @@ void StackLevelSetter::SetThrowHelperBlocks(GenTree* node, BasicBlock* block)
             SetThrowHelperBlock(SCK_ARITH_EXCPN, block);
         }
         break;
+        default:
+            // Other opers can target throw helper blocks only thought overflow exceptions.
```

Comment:
I don't understand this comment. Instead of "only thought" should it be "only due to"?

Reply:
Yes, I will change this assert and comment soon. Thanks.